### PR TITLE
Remove all hard coded strings

### PR DESCRIPTION
**Pull request by abs51295**
Fixes Hard coded lint issue of Issue #222.

### CODE REVIEW

File: app/src/main/res/layout/activity_circle_intro.xml [hunk 1]
```diff
@@ -25,8 +22,8 @@
         android:id="@+id/horizontal_line"
         android:layout_width="match_parent"
         android:layout_height="0.75dp"
-        android:alpha=".5"
         android:layout_above="@id/layoutDots"
+        android:alpha=".5"
```
Comment (sandarumk) [inline app/src/main/res/layout/activity_circle_intro.xml:26]: any reason for this?

File: app/src/main/res/layout/activity_circle_intro.xml [hunk 2]
```diff
@@ -36,33 +33,33 @@
         android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
         android:layout_alignParentRight="true"
-        android:background="@null"
-        android:src="@drawable/ic_navigate_next_white_24dp"
         android:layout_marginBottom="18dp"
-        android:layout_marginRight="28dp"/>
+        android:layout_marginRight="28dp"
+        android:background="@null"
```
Comment (sandarumk) [inline app/src/main/res/layout/activity_circle_intro.xml:38]: any reason for the displacement?

File: app/src/main/res/layout/activity_circle_intro.xml (hunk 1 above)
Comment (abs51295) [reply app/src/main/res/layout/activity_circle_intro.xml:26]: The displacement has occurred due to Auto format Code in Android Studio.

File: app/src/main/res/layout/activity_circle_intro.xml (hunk 2 above)
Comment (abs51295) [reply app/src/main/res/layout/activity_circle_intro.xml:38]: The displacement has occurred due to Auto format Code in Android Studio.

**Pull request merged**